config/routes.rb

```diff
@@ -7,4 +7,8 @@
 
   # Defines the root path route ("/")
   root "hothacks#index"
+
+  resources :data_vis
+  get "analytics" => "data_vis#data_view"
```

Comment:
Should we follow this route format for consistency or use eg:
get "/analytics", to: "data_vis#data_view"